en/docs/develop-components/develop-an-external-consumer.md

```diff
@@ -0,0 +1,98 @@
+# Develop an External Consumer
+
+An external consumer in Choreo is any client that can interact with services deployed in Choreo, as an entity hosted outside of the Choreo infrastructure. It is a generic component type that allows you to represent and manage any client, such as a mobile application, web application, server-to-server integration, bot, microservices, etc.
+
+In this guide, you will:
+
+ - Create an external consumer component.
+ - Manage authentication for the external consumer.
+ - Connect the external consumer to a service deployed in Choreo.
+ 
+## Prerequisites
+
+Before you try out this guide, complete the following steps:
+
+- If you are signing in to the Choreo Console for the first time, create an organization as follows:
+
+    1. Go to [https://console.choreo.dev/](https://console.choreo.dev/), and sign in using your preferred method.
+    2. Enter a unique organization name. For example, `Stark Industries`.
+    3. Read and accept the privacy policy and terms of use.
+    4. Click **Create**.
+
+    This creates the organization and opens the **Project Home** page of the default project created for you.
+
+## Step 1: Create an external consumer component
+ 
+To create an external consumer component, follow the steps given below:
+
+1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in. This opens the project home page.
+2. If you already have one or more components in your project, click **+ Create**. Otherwise, proceed to the next step.
+3. Click the **External Consumer** card. 
+4. In the **Create an External Consumer** pane that opens, enter a display name, a unique name, and a description for the component.
+5. Click **Create**. This creates the component and takes you to its **Overview** page.
+   
+## Step 2: Manage authentication for the external consumer
```

Comment:
@senthalan as discussed on the call need to see the exact steps we need to mention with regard to each  identity provider. Can you please have a look and let me know the steps to go with.